sonyflake_test.go

```diff
@@ -139,16 +137,17 @@ func TestSonyflakeInParallel(t *testing.T) {
 		go generate()
 	}
 
-	set := mapset.NewSet()
+	set := make(map[uint64]struct{})
 	for i := 0; i < numID*numGenerator; i++ {
 		id := <-consumer
-		if set.Contains(id) {
-			t.Fatal("duplicated id")
-		} else {
-			set.Add(id)
+		for k := range set {
+			if k == id {
```

Comment:
I think it's better to use  the “comma ok” idiom instead of the for loop:
```
if _, ok := set[id]; ok {
...
}
```

Reply:
@YoshiyukiMineo I fixed it. PTAL 🙇